src/test/resources/application.properties

```diff
@@ -16,4 +16,4 @@ spring.jpa.properties.hibernate.jdbc.time_zone=UTC
 cors.origin.allowed=*
 toleranceMultiplierInMinutes=3
 
-spring.jpa.properties.hibernate.metadata_builder_contributor=it.pagopa.interop.probing.eservice.operations.util.FunctionContributor
\ No newline at end of file
+spring.jpa.properties.hibernate.metadata_builder_contributor=it.pagopa.interop.probing.eservice.operations.util.FunctionContributorTest
```

Comment:
is there a reason to duplicate this code? shouldn't we test the actual `FunctionContributor` instead of the test version?

Reply:
You are right, but we are using an in memory db (HSQLDB) to test queries, which is different from the one we use for implementation (PostgreSQL). Since they are different databases, they don't all support the same functions. In this case the PostgreSQL db for extracting minutes from the difference between timestamp and lastrequest uses the `EXTRACT(EPOCH from (A-B))` function, instead HSQLDB uses the `DATEDIFF` function (I just pushed using this function for test cases) to do the same thing. The purpose of the tests is to test the real class, but also to test the correctness of the query itself. Here is the reason for using this test class.

Reply:
I agree on the purpose of testing the real class, but this is not actually testing the correctness of the query, because the tested query is not the real query in the real `FunctionContributor` class

Reply:
We opted for this solution to at least test the query logic, as the original function cannot be used on the in-memory db. If you don't agree with this solution, we should remove the test class entirely